src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -134,6 +134,26 @@ public void Add([NotNull] ITagHelper tagHelper)
         }
 
         /// <summary>
+        /// Tracks the minimized HTML attribute in <see cref="AllAttributes"/> and <see cref="HTMLAttributes"/>.
+        /// </summary>
+        /// <param name="name">The minimized HTML attribute name.</param>
+        public void AddHtmlAttribute([NotNull] string name)
```

Comment:
this will confuse everyone.  suggest either leaving out this "helper" or giving it a name that hints at minimization.